lib/helpers/answerHelper.js

```diff
@@ -116,12 +116,66 @@ function summaryFormatter(answerObject, fullUiSchema, answerLookup) {
     return answerIndex;
 }
 
+function objectFormatter(obj) {
+    const isArray = Array.isArray(obj);
+    let returnValue = '';
+    if (isArray) {
+        obj.forEach(answer => {
+            // eslint-disable-next-line no-use-before-define
+            const formattedAnswer = formatAnswerV2(answer);
+            returnValue += `${formattedAnswer}<br>`;
+        });
+        return returnValue.substring(0, returnValue.length - 4);
+    }
+    let lineEnd = '';
+    Object.keys(obj).forEach(answer => {
+        lineEnd = Object.keys(obj).length > 3 ? '<br>' : ' '; // For large numbers of fields, insert a line break.
+        if (obj[answer] !== '') {
+            // Ignore empty lines
+            // eslint-disable-next-line no-use-before-define
+            const formattedAnswer = formatAnswerV2(obj[answer]); // Format the individual answer.
+            lineEnd = formattedAnswer === 'Yes' || formattedAnswer === 'No' ? ', ' : lineEnd; // If a question was a boolean, follow it with a comma, not a space.
+            returnValue += `${formattedAnswer}${lineEnd}`; // append the answer and the line end to the return string.
+        }
+    });
+    return returnValue.substring(0, returnValue.length - lineEnd.length);
+}
+
+function dateFormatterV2(date, format) {
+    return moment(date).format(format);
+}
+
+function parseDateFormat(datePrecision) {
+    if (/^[yY\-/\s]*$/.test(datePrecision)) {
+        return 'YYYY';
```

Comment:
which of these formats is the most common? would it be worth having the most common one being evaluated first?

I did a quick perf test and found that the first condition evaluated twice as fast as the fall-through return

Reply:
`DD MMMM YYYY` is the most common but also the only one you would want to use as a fall through? We don't current require any dates be formatted as just the year, so I could remove this if it's an issue?